pd-control.md

```diff
@@ -312,16 +312,26 @@ config show cluster-version
     config set region-score-formula-version v2
     ```
 
-- `patrol-region-interval` 控制 replicaChecker 检查 Region 健康状态的运行频率，越短则运行越快，通常状况不需要调整。
+- `patrol-region-interval` 控制 checker 检查 Region 健康状态的运行频率，越短则运行越快，通常状况不需要调整。
 
-    设置 replicaChecker 的运行频率为 10 毫秒：
+    设置 checker 的运行频率为 10 毫秒：
 
     {{< copyable "" >}}
 
     ```bash
     config set patrol-region-interval 10ms
     ```
 
+- `patrol-region-worker-count` 控制 checker 检查 Region 健康状态时，创建 operator 的并发数，通常状况不需要调整。在当前版本中，将该配置项设置大于 1 时开启并发，为实验特性，不建议在生产环境中使用。
+
+    设置 checker 的并发数为2：
+
+    {{< copyable "" >}}
+
```

Comment:
```suggestion
```
This copy syntax is no longer needed for the current docs website. Please remove it from this doc.